src/test/java/core/basesyntax/service/impl/ParserServiceImplTest.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ParserServiceImplTest {
+    private ParserServiceImpl parser;

```

Comment:
Use interfaces instead of concrete implementations in your variable declarations.
ParserServiceImpl => ParserService

Reply:
I don't agree. In this case, we are testing a concrete implementation and cannot take another implementation.

Reply:
Agree with Anastasiia
```suggestion
    private ParserService parser;
```
Please fix here and in all similar cases in other test classes.